Assets/MRTK/SDK/Features/UX/Scripts/Buttons/ButtonConfigHelper.cs

```diff
@@ -4,13 +4,19 @@
 using TMPro;
 using UnityEngine;
 using UnityEngine.Events;
+using System;
+#if UNITY_EDITOR
+using UnityEditor;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+#endif
 
 namespace Microsoft.MixedReality.Toolkit.UI
 {
     /// <summary>
-    /// Helper component that gathers the most commonly modified button elemtents in one place.
+    /// Helper component that gathers the most commonly modified button elements in one place.
     /// </summary>
     [ExecuteAlways]
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Button.html#how-to-change-the-icon-and-text")]
```

Comment:
I wonder if there is a clean way to determine the correct doc version for the link? This one will always point to the dev branch docs.

This isnt anything to block this change upon, more an open question.